install/tests/Makefile

```diff
@@ -111,10 +111,14 @@ add-ns-record: check-env-cloud
 .PHONY:
 ## cluster-issuer: Creates a cluster issuer for the correspondign provider
 cluster-issuer: check-env-cloud
+ifndef self_signed
```

Comment:
you can do something like https://github.com/gitpod-io/gitpod/pull/12910/files#diff-bd7b7f3a397df93111a6b6e2a4284b01bd57e04474bc9f91f1b02ee1960399a2L163

Reply:
Updated to use `ifneq`

Reply:
But wondering if this means, `self-signed` is now triggered even when for `-a self-signed=false` (which I'm not sure anyone would do though). `ifneq` with a boolean seems to err for some reason. 🤔

Reply:
I feel like I might have been a bit short-sighted on my suggestion. I thought `self-signed` is either set or not since you were using `ifndef`. So, basically `self-signed` is a `string` that can have default value "false" or user provided value(can be "true", can be something else). So would it work if we did `ifneq($(self_signed),"false")` ?  
I promise I will unblock you soon :see_no_evil:

Reply:
So, using `ifneq` for boolean strings doesn't feel great. Instead, I'm now setting `self_signed` only when needed, and hence using `ifdef`. Trying that out now. Sorry for the back and forth!